waspc/src/Wasp/Psl/Generator/ConfigBlock.hs

```diff
@@ -0,0 +1,26 @@
+module Wasp.Psl.Generator.ConfigBlock
+  ( makeConfigBlockJson,
+  )
+where
+
+import Data.Aeson (KeyValue ((.=)), Value, object)
+import Data.Function (on)
+import Data.Functor ((<&>))
+import Data.List (nubBy)
+import qualified Wasp.Psl.Ast.Schema as Psl.Ast
+
+makeConfigBlockJson :: Psl.Ast.IsConfigBlock a => [(String, String)] -> a -> Value
+makeConfigBlockJson defaultValues configBlock =
+  object
+    [ "name" .= name,
+      "keyValues" .= (makeKeyValueJson <$> keyValues)
+    ]
+  where
+    name = Psl.Ast.getConfigBlockName configBlock
+    configBlockValues =
+      Psl.Ast.getConfigBlockKeyValues configBlock
+        <&> (\(Psl.Ast.ConfigBlockKeyValue key value) -> (key, value))
+    keyValues = nubBy ((==) `on` fst) (defaultValues ++ configBlockValues)
```

Comment:
Nice catch, I've renamed it to `overrideValues`.

As I've explained my reasoning, I think overriding is the way to go since they'll need to provide weird values that don't make sense when looking the `schema.prisma` file from their project root e.g. `file:./dev.db` value.

Reply:
Ok discussion is still ongoing there, I hope we will end with some comrpomise that does do overriding.
If you want we can resolve this issue, unless you need it as some kind of reminder for ir we go with overriding, but I doubt you do.